### ISSUE TITLE
NodeList is not defined

### ISSUE THREAD

**Issue by Bram-Zijp** [hirviid/react-redux-fetch#29]
I'm getting the following error:

```
ReferenceError: NodeList is not defined
    at A:\razzle-boilerplate\node_modules\iterators-polyfill\polyfill.min.js:1:2570
    at Object.<anonymous> (A:\razzle-boilerplate\node_modules\iterators-polyfill\polyfill.min.js:1:2977)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (A:\razzle-boilerplate\node_modules\react-redux-fetch\lib\index.js:6:1)
```

Using the Razzle redux example.

**Comment by hirviid** [comment]
Looks like Razzle doesn't like the Iterators polyfill and seamless-immutable. Looking into it!

**Comment by Bram-Zijp** [comment]
With your changes it works for me aswell.

**Comment by hirviid** [comment]
@Bram-Zijp Released version 0.12.6 !
